lib/kb_ea_utils/kb_ea_utilsImpl.py

```diff
@@ -502,7 +502,7 @@ def run_Fastq_Multx(self, ctx, params):
         except Exception as e:
             raise ValueError('Unable to get ReadsUtils Client' +"\n" + str(e))
         try:
-            if input_reads_obj_type == "KBaseFile.PairedEndLibarary":
+            if input_reads_obj_type == "KBaseFile.PairedEndLibrary":
```

Comment:
It did not fail in kb-sdk test, but did fail when I used a data object created using import in the narrative. Will add a test too.